packages/nimble-components/src/wafer-map/tests/data-generator.ts

```diff
@@ -56,7 +69,11 @@ export const generateWaferData = (
                 j--
             ) {
                 stringValue = generateStringValue(i, j, valueGenerator);
-                diesSet.push(generateDieContent(i, j, stringValue));
+                // Math.random() * (max-min) + min Generate a random number between two numbers
```

Comment:
What is the code in this comment referring to? If it's about the implementation of the generation function then it should go inside that function. But I think it's simple enough that it doesn't need to have a comment.

Reply:
My bad, it was there from the one previous implementation, I deleted it